Functions/README.md

```diff
@@ -7,17 +7,18 @@ integration test FirebaseFunctions:
 
 ### Prereqs
 
-- At least CocoaPods 1.6.0
-- Install [cocoapods-generate](https://github.com/square/cocoapods-generate)
+- Install [rvm](https://rvm.io/rvm/install)
+- `gem install bundle`
+- `bundle install`
 
 ### To Develop
 
-- Run `pod gen FirebaseFunctions.podspec`
+- Run `bundle exec pod gen FirebaseFunctions.podspec`
```

Comment:
Nit: We may revert `bundle exec` if we added `rvm` - `pod gen FirebaseFunctions.podspec` will do basically the same.